### DISCUSSION TITLE
Users list unpolulated must search + cannot delete categories

### DISCUSSION THREAD

**Discussion by adamelevate** [magicbell/community#100] [Questions] [answered]
First off, I love how easy this whole framework is, even starting with the Vue plugin is just amainzg.
 
When I load the users page, I see this error message:
`We're sorry. This should not have happened. Our engineers have been informed about the trouble.`

Also I'm unable to delete categories, the trash can icon doesn't do anything nor does it error.

Thanks again :)

**Comment by unamashana** [comment, marked as answer]
Hey @adamelevate apologies for the trouble, and for the late reply. The delete categories issue is fixed, and we also rolled out many improvements to user screens, so I'd request you to try again and please let us know if you still see the issue. 

Also, appreciate the words of encouragement. Thanks!
